metricbeat/module/ceph/osd_tree/_meta/fields.yml

```diff
@@ -21,6 +21,7 @@
       description: >
         osd or bucket node typeID
     - name: children
+      # TODO: Would prefer keyword here but potentially send up a list of all of them?
```

Comment:
I say we move it to keyword nonetheless.

And agree if this is typically a list of words, it would be beneficial to split them up before sending, but I don't consider this a blocker for fixing the datatype.

Reply:
I also fixed the code side to send up an array.

Reply:
And now the comment can be removed ;-)

Reply:
I still see the `children` field as text here. :confused-parrot:

Reply:
Updated now, thanks for catching.